README.md

```diff
@@ -1,3 +1,41 @@
 # Futbol
 
-Starter repository for the [Turing School](https://turing.io/) Futbol project.
+## Plan for Check-ins
+- Scheduled stand-ups (30ish minutes) each weekday (weekend schedule to be determined Friday at Retro)
+- Slack async check-ins every two hours/even hours 
+- Evening 9pm MT update/checkin, push work of the day to be ready for next standup
+- If working in morning, push by scheduled stand-up 
+
+## Plan for project organization
+- [Github Projects](https://github.com/users/DChavez18/projects/1)
+- [Miro board](https://miro.com/app/board/uXjVMBQe3Qk=/)
+- Building classes to organize data/hold state of objects, then methods to call/process that data
+- Mostly async, with pair/group when dealing with unproductive struggle 
+- Require 2 reviewers per pull request before merging to main 
+
+## Approaches to project organization
+We had a repeater in our group and we listened to his input on how his team previously used Miro board/GitHub projects. We were all more familiar with both GitHub and Miro board above the other options, too, and so went with the familiar. 
+We felt using GitHub Projects would be useful since we were already using GitHub to host our project, pull requests, comments, reviews, etc. 
+
+## Approach to code design
+We reviewed iteration 2/method requirements and used the Miro board to visualize what classes would be good choices to hold the right responsibilities to organize that data. We are planning to build class by class to first have objects to hold data/state, and then work on methods to access that data/behavior.
+
+## DTRs
+[Initial DTR, completed June 5th, 2023](https://docs.google.com/document/d/1lOCoLH7pUqNNJmmAIsmMe2ZGIiGfHvu5xhDQebHQWkE/edit)
+
+## Contributors
+### Garrett Varenhorst        
+[Github](https://github.com/GVarenhorst) | 
+[LinkedIn](https://www.linkedin.com/in/garrett-varenhorst-a734b466)
+
+### Artemy Gibson
+[Github](https://github.com/algibson1) |
+[LinkedIn](https://www.linkedin.com/in/artemy-gibson/)
+
+### Derek Chavez
+[Github](https://github.com/DChavez18) |
+[LinkedIn](https://www.linkedin.com/in/derek-chavez-1bb103276/)
+
+### Dani Wilson
+[Github](https://github.com/dani-wilson) |
```

Comment:
This looks great, Arty! I love the way you made our GitHub and LinkedIn links look so profesh!

Reply:
This looks very professional.